net/src/main/java/com/hiddenswitch/spellsource/impl/util/CollectionRecord.java

```diff
@@ -23,6 +23,7 @@
 	private boolean isStandardDeck;
 	private int wins;
 	private int totalGames;
+	private ValidationRecord validationRecord;
```

Comment:
if a `ValidationRecord` is a `ValidationReport` exactly, just use `ValidationReport`